spec/models/maneuver_participant_spec.rb

```diff
@@ -55,22 +55,26 @@
     end
   end
 
-  describe 'previous participant' do
-    it'finds previous participant' do
-      record_1 = create :maneuver_participant
-      record_2 = create :maneuver_participant
-      maneuver = create :maneuver,
-                        maneuver_participants: [record_1, record_2]
+  context 'previous participant' do
+    let!(:maneuver) { create :maneuver }
+    let!(:record_1)  { create :maneuver_participant, maneuver: maneuver }
```

Comment:
Very small thing, but looks like there are two spaces between the symbol and block argument to `let`.